src/view/use-sensor-marshal/is-event-in-interactive-element.js

```diff
@@ -43,6 +44,12 @@ function isAnInteractiveElement(parent: Element, current: ?Element) {
     return true;
   }
 
+  const dataAttributeName: string = interactiveElement.base;
+  const dataAttribute: ?string = current.getAttribute(dataAttributeName);
+  if (dataAttribute === 'true' || dataAttribute === '') {
```

Comment:
this feels right, but just want to question it. So we want it for either true or empty string? What if it is just `data-rbd-is-interactive-element` 🤔

Reply:
`data-rbd-is-interactive-element` by itself means data-rbd-is-interactive-element={true}, so I think it makes sense.

Empty string is questionable, but I opted for the same behavior as for the existing HTML contenteditable attribute.

Reply:
Good thinking. I will take a look at this next week

Reply:
@alexreardon Hi Alex. Got any time to review it? :)